### ISSUE TITLE
how to make the search result with pagination function

### ISSUE THREAD

**Issue by tonylu880042** [smartnsoft/flappy_search_bar#39]
Hi There,
I have a question, is anyway for me to do the pagination for the search results ? if the result comes for a large amount results.
Thanks for the reply.

**Comment by abhinavc** [comment]
I agree... Pagination should be supported. It's the only missing piece in this otherwise great package.

**Comment by sagreine** [comment]
not out of the box, but you could do it this way indirectly and likely improve on it a lot: 
https://github.com/smartnsoft/flappy_search_bar/issues/40#issuecomment-751559884
